core/src/test/scala/unit/kafka/tools/ConsumerPerformanceTest.scala

```diff
@@ -0,0 +1,59 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.kafka.tools
+
+import java.io.ByteArrayOutputStream
+import java.text.SimpleDateFormat
+
+import kafka.tools.ConsumerPerformance
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+
+class ConsumerPerformanceTest {
+
+  private val outContent: ByteArrayOutputStream = new ByteArrayOutputStream()
+
+  @Test
+  def testPrintProgressMessage(): Unit = {
+    Console.withOut(outContent) {
+      ConsumerPerformance.printProgressMessage(1, 1024 * 1024, 0, 1, 0, 0, 1,
+        new SimpleDateFormat("yyyy")
+      )
+    }
+    val expected = "1970, 1, 1.0000, 1000.0000, 1, 1000.0000"
+    assertEquals(expected, outContent.toString.trim)
```

Comment:
This test may fail depending on the configured timezone (it failed on my laptop), so I'll remove it.